ui/components/Filters.js

```diff
@@ -1470,13 +1462,13 @@ const ImportModal = React.memo((props) => {
 });
 
 const PublishModal = React.memo((props) => {
-  const { publishFormSchema, handleClose, handlePublish, title } = props;
+  const { handleClose, handlePublish, title } = props;
 
   return (
     <Modal
       open={true}
-      schema={publishFormSchema.rjsfSchema}
-      uiSchema={publishFormSchema.uiSchema}
+      schema={publishCatalogItemSchema}
+      uiSchema={publishCatalogItemUiSchema}
```

Comment:
Thankyou for updating @sudhanshutech 's PR changes too, however here do make sure you are using updating schema that you updated [here](https://github.com/meshery/meshery/pull/10720/files#diff-6ec8e150ffa5db5b0830cd1598d529bbc98c79c98df32092858a51a779fcdea0R355)
```suggestion
      schema={publishFormSchema.rjsfSchema}
      uiSchema={publishFormSchema.uiSchema}
```